Source/ProgressionSystemRuntime/Private/ProgressionSystemRuntimeModule.cpp

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#include "ProgressionSystemRuntimeModule.h"
+
+#define LOCTEXT_NAMESPACE "FProgressionSystemRuntimeModule"
+
+void FProgressionSystemRuntimeModule::StartupModule()
+{
+	// This code will execute after your module is loaded into memory;
+	// the exact timing is specified in the .uplugin file per-module
+}
+
+void FProgressionSystemRuntimeModule::ShutdownModule()
+{
+	// This function may be called during shutdown to clean up your module.
+	// For modules that support dynamic reloading, we call this function before unloading the module.
+}
+
+#undef LOCTEXT_NAMESPACE
+
+IMPLEMENT_MODULE(FProgressionSystemRuntimeModule, ProgressionSystemRuntime)
```

Comment:
This `IMPLEMENT_MACRO` requires `#include "Modules/ModuleManager.h"` at the top. Right now, this file fails to compile on Mac platform.

To reproduce this issue, from now please always compile in non-unity mode:
1. Go to `%appdata%\Unreal Engine\UnrealBuildTool\`.
2. Open existing `BuildConfiguration.xml` or create new one if not found.
3. Replace all by next:
```
<?xml version="1.0" encoding="utf-8" ?>
<Configuration xmlns="https://www.unrealengine.com/BuildConfiguration">
  <BuildConfiguration>
	<bUseUnityBuild>false</bUseUnityBuild>
	<bUsePCHFiles>false</bUsePCHFiles>
  </BuildConfiguration>
</Configuration>
```
4. Compile project
5. Notice, your module fails to compile. Fix the issue, add mentioned include.

By default, all UE projects have enabled Unity builds to speed up compilation time by merging multiple source files into one. However, it hides dependencies between files, leading to such issues that only surface when the unity build is disabled.

Reply:
solved https://github.com/h4rdmol/ProgressionSystem/pull/1/commits/aaf19939eb4c1582f79acbe02292616d28ec83df

Reply:
@h4rdmol Is not finished. Compile error fixed, but why included was added to .h, but not to .cpp. `IMPLEMENT_MODULE` that needs include is in .cpp by itself. Please move include

Reply:
migrated https://github.com/h4rdmol/ProgressionSystem/pull/1/commits/28d0c6bed2bf8963d1ce825d371895a115e61671

Is there a difference in where to keep it?

Reply:
Any .h should have as less includes as possible to decrease compile time as it will be less files compiling when you change anything in included header